pelican/plugins/webassets/webassets.py

```diff
@@ -14,7 +14,7 @@
 .. _webassets: https://webassets.readthedocs.org/
 
 """
-from __future__ import unicode_literals
+# from __future__ import unicode_literals
```

Comment:
Feel free to remove that `import` entirely: this plugin is [no longer Python 2.x compatible](https://github.com/pelican-plugins/webassets/blob/856cfa4b5d8cfeee5916909dc480ccbff737d204/pyproject.toml#L37).

Reply:
Thanks, I totally didn't notice that! I've appended this to the c51b978 **- make poetry's linters happy** commit